ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java

```diff
@@ -12,21 +12,23 @@
 import java.util.List;
 
 import io.syndesis.qe.pages.SyndesisPageObject;
-import io.syndesis.qe.pages.integrations.edit.steps.StepFactory;
-import io.syndesis.qe.pages.integrations.edit.steps.StepPage;
+import io.syndesis.qe.pages.integrations.edit.steps.StepComponentFactory;
+import io.syndesis.qe.pages.integrations.edit.steps.StepComponent;
+
+public class IntegrationFlowViewComponent extends SyndesisPageObject {
 
-public class FlowViewComponent extends SyndesisPageObject {
-	
-	  
 	private static final class Element {
 		public static final By ROOT = By.cssSelector("syndesis-integrations-flow-view");
 
 		public static final By NAME = By.cssSelector("input.form-control.integration-name");
-		public static final By STEP = By.cssSelector("parent-step");
+		public static final By STEP = By.cssSelector("div.parent-step");
 		public static final By ACTIVE_STEP = By.cssSelector("div[class='parent-step active']");
-		public static final By PARENT_STEP = By.cssSelector("div.parent-step");
+		//DELETE AND DELETE ARE IDENTICAL
```

Comment:
> //DELETE AND DELETE ARE IDENTICAL 

???

Reply:
sorry, it should be: DELETE and TRASH are identical, they refered to the same element.